akka-docs/src/main/paradox/stream/operators/Source-or-Flow/alsoToAll.md

```diff
@@ -0,0 +1,31 @@
+# alsoToAll
+
+Attaches the given `Sink`s to this `Flow`, meaning that elements that pass through this `Flow` will also be sent to all those [[Sink]]s.
+
+@ref[Fan-out operators](../index.md#fan-out-operators)
+
+## Signature
+
+@apidoc[Source.alsoToAll](Source) { scala="#alsoToAll(that:akka.stream.Graph[akka.stream.SinkShape[Out],_]*):FlowOps.this.Repr[Out]" java="#alsoTo(akka.stream.Graph)" }
+@apidoc[Flow.alsoToAll](Flow) { scala="#alsoToAll(that:akka.stream.Graph[akka.stream.SinkShape[Out],_]*):FlowOps.this.Repr[Out]" java="#alsoTo(akka.stream.Graph)" }
```

Comment:
This links to `alsoTo` for Java, shouldn't that also be `alsoToAll`?

Reply:
Will fix